packages/git-town/build.sh

```diff
@@ -0,0 +1,33 @@
+TERMUX_PKG_DESCRIPTION="Git branches made easy"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="21.5.0"
+TERMUX_PKG_HOMEPAGE="https://www.git-town.com"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
```

Comment:
```suggestion
```
The license file gets picked up automatically if it's a standard name.
https://github.com/termux/termux-packages/blob/94dc383d40f6459265a24bed22e234c54b97aa95/scripts/build/termux_step_install_license.sh#L36-L44